src/test/datascience/export/fileConverter.vscode.web.test.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+'use strict';
+
+import { anything, instance, mock, verify, when } from 'ts-mockito';
+import * as sinon from 'sinon';
+import { Uri } from 'vscode';
+import { IApplicationShell } from '../../../platform/common/application/types';
+import { IConfigurationService, IWatchableJupyterSettings } from '../../../platform/common/types';
+import { ExportFileOpener } from '../../../platform/export/exportFileOpener';
+import { FileConverter } from '../../../platform/export/fileConverter.web';
+import { IExport, IExportDialog, ExportFormat } from '../../../platform/export/types';
+
+suite('DataScience - File Converter', () => {
+    let fileConverter: FileConverter;
+    let exportPythonPlain: IExport;
+    let filePicker: IExportDialog;
+    let appShell: IApplicationShell;
+    let exportFileOpener: ExportFileOpener;
+    let configuration: IConfigurationService;
+    let settings: IWatchableJupyterSettings;
+    setup(async () => {
+        filePicker = mock<IExportDialog>();
+        exportPythonPlain = mock<IExport>();
+        appShell = mock<IApplicationShell>();
+        exportFileOpener = mock<ExportFileOpener>();
+        configuration = mock<IConfigurationService>();
+        settings = mock<IWatchableJupyterSettings>();
+        when(configuration.getSettings(anything())).thenReturn(instance(settings));
+        when(settings.pythonExportMethod).thenReturn('direct');
+        when(filePicker.showDialog(anything(), anything(), anything())).thenReturn(
+            Promise.resolve(Uri.file('test.py'))
+        );
+        when(appShell.showErrorMessage(anything())).thenResolve();
+        when(exportPythonPlain.export(anything(), anything(), anything())).thenResolve();
+        when(filePicker.showDialog(anything(), anything())).thenResolve(Uri.file('foo'));
+        when(exportFileOpener.openFile(anything(), anything(), anything())).thenResolve();
+        fileConverter = new FileConverter(
+            instance(exportPythonPlain),
+            instance(filePicker),
+            instance(exportFileOpener)
+        );
+    });
+    teardown(() => sinon.restore());
+
+    test('Export to Python is called when export method is Python', async () => {

```

Comment:
I know this is just the same thing as the existing vscode test. But looking at those tests, I'm not really sure they buy us much? In any case I have no issue with adding it.

Reply:
Just rethinking those existing tests myself. Feels like they are just confirming some internal implementation and that's it.

Reply:
Oh yeah this test is rather simple. I was hoping there was a test that opened a notebook and exported it, verifying the text got generated. 

This is just testing the internals of the fileConverter.